README.md

```diff
@@ -64,7 +64,7 @@ Note - extra features are NOT guaranteed to be maintained in future releases, pa
 
 Modify config.yml:
 
- - hostname: * - the interface to bind to. Can be "localhost", "\*", an IP address or a hostname.
+ - hostname: 0.0.0.0 - the interface to bind to. Can be "localhost", "0.0.0.0" (all interfaces), an IP address or a hostname.
```

Comment:
In retrospect I wonder whether the default included in the config.yml should be blank? That way the default operation keeps it as is. 

```
- hostname: 
```

I have a minor worry about security in windows `0.0.0.0`. I have seen it cause problems in the past where users don't have permission to bind to it.

Reply:
The „0.0.0.0“ doesn‘t get passed to the OS, it gets recognized and InetSocketAddress without hostname is used